### DISCUSSION TITLE
[Help] - XTTS can only generate text with a maximum of 400 tokens.

### DISCUSSION THREAD

**Discussion by johndjchan** [coqui-ai/TTS#4080] [General Q&A]
i successfully trained with the xTTs model, but it keep saying error of  XTTS can only generate text with a maximum of 400 tokens.
which is very short?

is there a way to increase the limit?

//
Warning: The text length exceeds the character limit of 250 for language 'en', this might cause truncated audio.

**Comment by eginhard** [comment]
You can synthesise longer text by splitting it into sentences and processing them separately.

(you can technically also increase the limit, but as the warning says it will just result in bad outputs because the model is not trained on longer data)

**Reply by johndjchan** [reply]
thanks a lot  for tkaing the time to reply me
i managed to use splitting but the rendering became very long
muchlonger than one audio one as it need combination of the audio files
